GWAS/Region_Extraction.ipynb

```diff
@@ -151,25 +151,25 @@
     "# Work directory where output will be saved to\n",
     "parameter: cwd = path\n",
     "# Region specifications\n",
-    "parameter: region_file = path\n",
+    "parameter: region_file = path()\n",
```

Comment:
When using `path` and not `path()`, we require that the input must be provided. is there a specific reason that you have to use `path()` instead?